filters/AssignFilter.cpp

```diff
@@ -63,14 +63,14 @@ void AssignRange::parse(const std::string& r)
     char *end;
 
     pos = subParse(r);
-    count = Utils::extractSpaces(r, pos);
+    count = Utils::extract(r, pos, (int(*)(int))std::isspace);
```

Comment:
The changes to AssignFilter seem out of place. This actually reverts a commit from four months ago. Probably just need to rebase with master?